specs/experimental/cannon-fault-proof-vm-mt.md

```diff
@@ -224,24 +236,36 @@ FPVM state to track memory reservations initiated by `ll` operations.
 
 When an `ll` instruction is executed:
 
-- `llReservationActive` is set to true.
+- `llReservationStatus` is set to `1`.
 - `llAddress` is set to the memory address specified by `ll`.
 - `llOwnerThread` is set to the `threadID` of the active thread.
 
 Only a single memory reservation can be active at a given time - a new reservation will clear any previous reservation.
 
 When the VM writes any data to memory, these `ll`-related fields are checked and the memory reservation is cleared if
-a memory write touches a reserved `llAddress`.
+a memory write touches a `Word` that contains `llAddress`.
 
 When an `sc` instruction is executed, the operation will only succeed if:
 
-- There exists an active reservation (`llReservationActive == true`).
+- There exists an active 4-byte memory reservation (`llReservationStatus == 1`).
 - The active thread's `threadID` matches `llOwnerThread`.
 - The requested address matches `llAddress`.
 
-On success, `sc` stores a value at the target memory address, clears the memory reservation and returns `1`.
+On success, `sc` stores a value at the target memory address, clears the memory reservation by zeroing out
+`llReservationStatus`, `llOwnerThread`, and `llAddress` and returns `1`.
 On failure, `sc` returns `0`.
 
+### Load Linked / Store Conditional Doubleword
+
+With the transition to MIPS64, Load Linked Doubleword (`lld`), and Store Conditional Doubleword (`scd`) instructions
+are also now supported.
+These instructions are similar to `ll` and `sc`, but target 8-byte rather than 4-byte memory segments.
```

Comment:
Note that in Cannon, both `ll` and `lld` "target 8-byte memory segments" since the reservation is invalidated when any byte located at its effective address (which spans an 8-byte word on MIPS64) is written to.

Reply:
Just cut this line altogether because it's confusing and doesn't really add much.